src/runtime/manifest.ts

```diff
@@ -1362,6 +1362,27 @@ ${e.message}
     return recipe;
   }
 
+  // TODO: This is a temporary method to allow sharing stores with other Arcs.
+  registerStore(store: UnifiedStore, tags: string[]): void {
+    if (!this.findStoreById(store.id) && tags.includes('shared')) {
+      this['_addStore'](store, tags);
```

Comment:
since this is a Manifest method now, you could just use `this._addStore`directly

Reply:
Thanks, done.